### PR TITLE
Propagation of response.encoding

### PR DESCRIPTION
**Pull request by miro-balaz**
As per http://www.softwareishard.com/blog/har-12-spec/#content , there is "encoding" field in content. I need to be able to use it, because in my project I am base64 encoding certain reponses.

### CODE REVIEW

**Comment by soulgalore** [comment]
Hi @miro-balaz aha cool. I think you can fix the format issues with` npm run lint:fix`.

**Comment by miro-balaz** [comment]
I think it should be ok now, i just commited it into my branch and it automatically appeared in PR, i didn't expect that.

File: lib/entryFromResponse.js [hunk 1]
```diff
@@ -79,8 +79,9 @@ module.exports = function(entry, response, page, options) {
     status: response.status,
     statusText: response.statusText,
     content: {
+      encodig: response.encoding,
```
Comment (beatchristen) [inline lib/entryFromResponse.js:82]: This would be really useful to have!
May I suggest to have the property corrected to `encoding`?

Comment (beatchristen) [reply lib/entryFromResponse.js:82]: See https://github.com/beatchristen/chrome-har/commit/20b078ef0d720df8b784c906faa555b203b0097b for my variant, which also tries to adhere to the 'optionality' aspect of the property.

**Comment by soulgalore** [comment]
Oopss, I totally missed this PR, I'll merge and change the field name.

**Pull request merged**